shared/src/vyxal/VNum.scala

```diff
@@ -55,11 +55,31 @@ class VNum private (val underlying: Complex[Real]) extends Ordered[VNum]:
     this - spire.math.floor(q.real) * rhs
 
   def vabs: VNum = underlying.abs
+  def arg: VNum = underlying.arg
 
   /** Inclusive range */
   def to(end: VNum, step: VNum = 1): VList =
     VList.from(Inclusive(this, end, step))
 
+  def sin: VNum = underlying.sin
+  def cos: VNum = underlying.cos
+  def tan: VNum = underlying.tan
+  def asin: VNum = underlying.asin
+  def acos: VNum = underlying.acos
+  def atan: VNum = underlying.atan
+  // def atan2(rhs: VNum): VNum = spire.math.atan2(underlying, rhs.underlying)
```

Comment:
Should probably leave a "TODO" here along with a link to https://github.com/typelevel/spire/issues/1313 so we can come back to this when the Spire bug is fixed